sdcm/utils/decorators.py

```diff
@@ -238,3 +238,15 @@ def __get__(self, instance, owner=None):
             return super().__get__(instance=instance, owner=owner)
         except NoValue:
             return None
+
+
+def static_init(cls):
+    """A decorator for classes that make sure to call 'static_init' method if
+    such a method exists.
+
+    Returns:
+        The class that was initialized.
+    """
+    if getattr(cls, "static_init", None):
```

Comment:
It could be `if hasattr(cls, "static_init"):`

Reply:
Will change